app/queries/developer_query.rb

```diff
@@ -0,0 +1,81 @@
+class DeveloperQuery
+  include Pagy::Backend
+
+  alias_method :build_pagy, :pagy
+
+  attr_reader :options
+
+  def initialize(options = {})
+    @options = options
+  end
+
+  def available?
+    return nil if options[:available] == ""
```

Comment:
```suggestion
    return nil if options[:available].blank?
```

Will account for empty string and `nil`.